.github/ISSUE_TEMPLATE/copy.yml

```diff
@@ -0,0 +1,41 @@
+name: "Copy"
+description: Additions or updates to the copy
+title: "🔤 "
+projects: ["GCTC-NTGC/8"]
+type: "Debt"
```

Comment:
I suggest copy  have a default type of either feature or bug since it is visible to the users.